src/api/services/FriendRequestApi.js

```diff
@@ -0,0 +1,41 @@
+import BASE_URL from "./BaseUrl";
+const axios = require("axios");
+const userToken = localStorage.getItem("token");
+
+export const allUSers = async () => {
+  let config = {
```

Comment:
This config object is created in each and every API call, so make a common method for all APIs have common steps